src/ResourceManager/EventGrid/Commands.EventGrid/help/AzureRM.EventGrid.md

```diff
@@ -43,3 +40,7 @@ Removes an Azure Event Grid Topic.
 
 ### [Set-AzureRmEventGridTopic](Set-AzureRmEventGridTopic.md)
 Sets the properties of an Event Grid topic.
+
+### [Update-AzureRmEventGridSubscription](Update-AzureRmEventGridSubscription.md)
```

Comment:
Seems like the help files weren't updated with the new parameters - please regenerate.